alert-common/src/main/java/com/blackduck/integration/alert/common/AlertProperties.java

```diff
@@ -189,6 +192,7 @@ public Long getLoginLockoutMinutes() {
     public Optional<Integer> getNotificationMappingBatchLimit() {
         return Optional.ofNullable(notificationMappingBatchLimit);
     }
+    public Optional<Integer> getAccumulatorNotificationBatchLimit() {return Optional.ofNullable(accumulatorNotificationBatchLimit);}
```

Comment:
Formatting nitpick: the return should be on a separate line like in other getters in this class